src/presets.asm

```diff
@@ -232,18 +231,15 @@ preset_to_memory:
 
   .two
     LDA ($00),Y : INY : INY : STA [$C3]
-    INX #6
     BRA .loop
 
   .one
     %a8()
     LDA ($00),Y : INY : STA [$C3]
     %a16()
-    INX #5
     BRA .loop
 
   .done
-    PLX
```

Comment:
I haven't tested the changes to preset_to_memory yet, but just on code inspection, I don't see why this method is messing with the X register.